.github/workflows/docker-publish.yml

```diff
@@ -1,52 +1,34 @@
-name: Docker
-
+name: Docker Publish
 on:
   push:
     branches: [main]
-    # Publish semver tags as releases.
     tags: ["v*.*.*"]
-  pull_request:
-    branches: [main]
-
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: ${{ github.repository }}
-
 jobs:
   build:
     runs-on: ubuntu-latest
     permissions:
       contents: read
       packages: write
-
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
```

Comment:
Why? I think this is easier to read and gives more info about what was exactly run:

![image](https://user-images.githubusercontent.com/16060539/147934086-e503acb8-15f1-4cc6-a7f0-c2dac6c41cc4.png)

Reply:
If one knows what the cargo commands do yes, but we are an open source repository, and therefore we want to be as clear to possible new contributors as possible.

Reply:
Cargo was an example. Just saying that the names add little to nothing, imo. Like "Docker build" instead of "docker-build/action" is pretty much the same, even to a newcomer. But I will then for consistency just add names on all.

Reply:
But cargo was a good example, as not everyone is familiar with its workings. At least in the beginning.